keras/src/backend/openvino/excluded_concrete_tests.txt

```diff
@@ -9,7 +9,6 @@ NumpyOneInputOpsCorrectnessTest::test_argmin
 NumpyOneInputOpsCorrectnessTest::test_argpartition
 NumpyOneInputOpsCorrectnessTest::test_argsort
 NumpyOneInputOpsCorrectnessTest::test_array
-NumpyOneInputOpsCorrectnessTest::test_average
```

Comment:
please also include NumPyDType test for average. We just recently merge PR for this.

https://github.com/keras-team/keras/pull/20929/files#diff-60b6b6e36719bf6926e4fcd3ca37f40d5b2ab11e55a3d4b3f40e79a092d64d23R15